2023/Ihedioha-Chinedu/README.md

```diff
@@ -54,26 +54,54 @@ And now, we will want you to start using GitHub so we can review your progress o
 - [ ] With your access to the demo site and by reviewing our [user guide](https://guide.wagtail.org/), identify a feature of Wagtail which isn’t documented, and report it back on the [guide project issue tracker](https://github.com/wagtail/guide/issues). Make sure to explain what the feature is, why and how you think it should be documented.
 - [ ] Draft the new content to document the feature of Wagtail which you’ve reported isn’t documented. Add it to the issue you created as a new comment, so we can provide feedback.
 - [ ] Ask [@thibaudcolas](https://github.com/thibaudcolas) via email or on Slack to give you editor access to [guide.wagtail.org](https://guide.wagtail.org/). Once you’ve received access, we ask you to make the necessary changes in the CMS to add the feature documentation.
 - [ ] TBC – Additional tasks specific to the Outreachy application process.
 
-## Research
\ No newline at end of file
+## Research
+
+🚧 This is the list of links to the  resources I used in trying to understand Wagtail as a project and project ideas:
+
+- [What is Wagtail CMS and How does it Work?](https://www.tekkiwebsolutions.com/blog/what-is-wagtail-cms/)
+- [How to Use the Wagtail CMS for Django: An Overview](https://steelkiwi.com/blog/how-to-use-the-wagtail-cms-for-django-an-overview/)
+- [Wagtail vs Wordpress](https://wagtail.org/wagtail-vs-wordpress/)
+- [wagtail.org](https://wagtail.org/)
+- [Accessibility features documentation](https://github.com/wagtail/outreachy/blob/main/project-ideas.md#accessibility-features-documentation)
+
+🚧 These are the list of recent articles I have read and liked:
+
+- [Beginner's guide to open-source contribution](https://workat.tech/general/article/open-source-contribution-guide-xmhf1k601vdj#:~:text=What%20do%20you%20mean%20by,improvement%20of%20open-source%20software.)
+- [How to Contribute to Open Source Projects](https://www.freecodecamp.org/news/how-to-contribute-to-open-source-projects-beginners-guide/)
+- [Spring Boot Project Folder Structure](https://dev.to/jazzybruno/spring-boot-project-folder-structure-12oe)
+- [Entity Relationship (ER) Diagram Model with DBMS Example](https://www.guru99.com/er-diagram-tutorial-dbms.html)
+- [SOLID Principles In Java](https://medium.com/@ersin.yilmaz.aslan/solid-principles-in-java-f27b744905f7)
+
+## My blog post on Open-Source Contribution
+
+🚧 I recently published a blog post on my recent learnings in the field of open-source contribution. Check it out on [Hashnode](https://chinexboroja24.hashnode.dev/embarking-on-a-new-journey-in-open-source-contribution)
+
+## My blog post on Accessibility
+
+🚧 I published a blog post on understanding web accessibility. This blog talks about ensuring an inclusive online experience for all users, regardless of any health problems. You can read the blog post on [Hashnode](https://chinexboroja24.hashnode.dev/understanding-web-accessibility)
```

Comment:
I see nothing on there about your own experience / understanding, just very generic information.

Reply:
Ok noted @thibaudcolas , I will try and optimize the post